jina/resources/k8s/template/deployment-init.yml

```diff
@@ -37,13 +37,9 @@ spec:
           - containerPort: {port_in}
           - containerPort: {port_out}
           - containerPort: {port_ctrl}
-        env:
-        - name: JINA_LOG_LEVEL
-          value: debug
-        - name: PYTHONUNBUFFERED
-          value: "1"
-        - name: WORKER_CLASS
-          value: "uvicorn.workers.UvicornH11Worker"
+        envFrom:
+          - configMapRef:
+              name: config-map
```

Comment:
If you have multiple executors within the same namespace, they will overwrite the config-maps. Therefore, they need a different name, e.g., {executor_dns_name}-env.